perf/kernbench.py.data/config

```diff
@@ -0,0 +1,2980 @@
+# Kernel Configuration
```

Comment:
can we use running kernel config file which reside /boot/config* 

python code can be 
```
import platform
-------------
kernel_version = platform.uname()[2]
config_file = '/boot/config-' + kernel_version
-------------- 
```